homeassistant/components/mhtzn/__init__.py

```diff
@@ -0,0 +1,69 @@
+"""The Detailed MHTZN integration."""
+from __future__ import annotations
+
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .Gateway import Gateway
+from .const import PLATFORMS, MQTT_CLIENT_INSTANCE, CONF_LIGHT_DEVICE_TYPE, DOMAIN, FLAG_IS_INITIALIZED, \
+    CACHE_ENTITY_STATE_UPDATE_KEY_DICT
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def _async_config_entry_updated(hass: HomeAssistant, entry: ConfigEntry) -> None:
+    """This method is triggered when the entry configuration changes, and the gateway link information is updated"""
+
+    """Get gateway instance"""
```

Comment:
Don't use docstring as comments please.

Reply:
Thanks for the tip, I have deleted the note, please review it again

Reply:
> Don't use docstring as comments please.

Thank you for your hint, I have changed it to English, please review it again

Reply:
All update request processed